pkg/config/config.go

```diff
@@ -289,7 +289,7 @@ func InitConfig(config Config) {
 	config.BindEnvAndSetDefault("remote_configuration.director_root", "")
 	config.BindEnvAndSetDefault("remote_configuration.refresh_interval", 1*time.Minute)
 	config.BindEnvAndSetDefault("remote_configuration.max_backoff_interval", 5*time.Minute)
-	config.BindEnvAndSetDefault("remote_configuration.clients.ttl_seconds", 30*time.Second)
+	config.BindEnvAndSetDefault("remote_configuration.clients.ttl_seconds", 30)
```

Comment:
All the other configs are durations, let's keep it consistent one way or the other

Reply:
Fixed, also fixed the reporting to properly display the time duration instead of integers